app/src/main/java/com/kks/nimblesurveyjetpackcompose/model/entities/Survey.kt

```diff
@@ -0,0 +1,13 @@
+package com.kks.nimblesurveyjetpackcompose.model.entities
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+@Entity(tableName = "survey")
+data class Survey(
+    @PrimaryKey
+    val id: String,
```

Comment:
Don't forget to define column names, otherwise the database won't work properly after obfuscation 😉 
```suggestion
    @ColumnInfo(name = "id") val id: String,
```